pkg/apps/utils.jsx

```diff
@@ -43,12 +43,18 @@ export function icon_url(path_or_url) {
     if (path_or_url[0] != '/')
         return path_or_url;
 
+    var contentType = ""
+    if (path_or_url.endsWith("svg")) {
+         contentType = "image/svg+xml"
+    } else {
+         contentType = "image/png"
+    }
     var query = window.btoa(JSON.stringify({
         payload: "fsread1",
         binary: "raw",
         path: path_or_url,
         external: {
-            "content-type": "image/png",
+            "content-type": contentType,
```

Comment:
Can we let the browser do the content type detection, actually?

Reply:
I tried removing the `"content-type"` entirely, and PNG icons are still displayed correctly. So I vote to remove it.  Would that work for you as well, @jeepingben?

Reply:
How about if we only set content-type if we are loading svgs?